### ISSUE TITLE
MS Teams not supported?

### ISSUE THREAD

**Issue by pawel221** [bdkjones/MuzzleApp#86]
Hi

Nice idea to create such app. Recently I had event when I was sharing screen and got unwanted message. Therefore I would use your app but it seems it doesn't support MS Teams, at least on my machine. I tested screen recoding with QuickTime and Muzzle worked as expected (except it didn't turn back on the do not disturb mode, after recording was finished).

Can you add MS Teams support.

I'm using Catalina 10.15.6 (19G2021)
Muzzle version 1.7 Build 405
MS Teams - latest version I can not find the number.
I see Muzzle in the Security&Privacy->Privacy->Accessibility list and it's checked

Thanks
Paweł

**Comment by BartoszRadaczynski-TomTom** [comment]
duplicate of https://github.com/bdkjones/MuzzleApp/issues/76

**Comment by radaczynski** [comment]
https://github.com/bdkjones/MuzzleApp/issues/76

**Comment by x74353** [comment]
Also had this issue with Teams. Notifications were not silenced when sharing my screen via Teams. macOS 11.2 (20D64).

**Comment by GentleHoneyLover** [comment]
+1 for MS Teams support!

**Comment by cfarvidson** [comment]
+1 for MS Teams support!

**Comment by radaczynski** [comment]
@bdkjones  - I think it's fine to close tickets with comment "MS should just use the correct API", but do you know of a way to give them this feedback (e.g. via their uservoice or some other way)?

**Comment by adib** [comment]
Yeah, MS Teams is not supported as of Monterey 12.0.1 & Teams 1.4.00.22368. I've tested by screen sharing and making a Facetime audio call to myself. These two notifications are consistently displayed

1. "... is calling" notification
2. Missed call notificstaion.

**Comment by muslimsteel** [comment]
There is a setting in Teams that lets you change it to use the Mac notification system, which I did and it now complies with the restrictions that are in place for other notifications:
<img width="780" alt="Screen Shot 2022-03-14 at 3 02 21 PM" src="https://user-images.githubusercontent.com/28244480/158242725-0154ed38-696e-49aa-a1c3-6040720a537b.png">

**Comment by adib** [comment]
The _Notifications_ setting in Teams **does not** solve the problem of notifications being shown while Teams is sharing a screen. There is another configuration in System Preferences, Notifications & Focus, the _When mirroring or sharing the display_ checkbox. Unfortinately Teams screen sharing seems to ignore this.

**Comment by keiohtani** [comment]
+1
